spec/porky_lib/file_service_spec.rb

```diff
@@ -90,7 +90,7 @@ def stub_large_file
   def write_test_file(data)
     tempfile = Tempfile.new
     tempfile << data
-    tempfile.close
+    tempfile.flush
```

Comment:
`close` has been changed to `flush` since `File.size()` of the tempfile would throw an error. Both do the same thing, they both write to a file and at the end of the tests, the files get deleted